_topic_map.yml

```diff
@@ -1377,8 +1377,8 @@ Topics:
       File: cnv-attaching-vm-multiple-networks
     - Name: Installing the QEMU guest agent on virtual machines
       File: cnv-installing-qemu-guest-agent
-    - Name: Viewing the IP address of vNICs on a virtual machine
-      File: cnv-viewing-ip-of-vm-vnic
```

Comment:
If a file name for an assembly is changing, we will need to add redirects as well so folks don't get 404s. @lmandavi when you are ready, can you reach out to me about it?